packages/eslint-config-airbnb-base/rules/best-practices.js

```diff
@@ -45,11 +46,11 @@ module.exports = {
 
     // encourages use of dot notation whenever possible
     // https://eslint.org/docs/rules/dot-notation
-    'dot-notation': ['error', { allowKeywords: true }],
+    'dot-notation': ['warn', { allowKeywords: true }],
 
     // enforces consistent newlines before or after dots
     // https://eslint.org/docs/rules/dot-location
-    'dot-location': ['error', 'property'],
```

Comment:
Reverse and leave as `error`

Reply:
Will be introduced as a go-forward hard standard and IF NEEDED will allow a override on a legacy project.

Reply:
Reversed.